deploy/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+# requires dhtnode docker image already present in the cache
+# start with `docker-compose up dhtnode`
+#
+# after that run `docker-compose run shell` to
+# get into the interactive shell with an access to all running services.
+version: '2'
+services:
+  dhtnode:
+    image: sociomantictsunami/dhtnode
+    ports:
```

Comment:
As discussed in the chat, there is no need to publish the ports when using docker-compose.

Reply:
Done. Also changed listening address in config.